### ISSUE TITLE
Universal Buttons Panel - poor text contrast -  1.4.3 Contrast (Minimum)

### ISSUE THREAD

**Issue by syarwick23** [stephenravenscraft/loyola-template-4.0#113]
for buttons with a gold background, all the buttons appear as black text on a maroon background, whihc does not have a sufficient contrast ratio of 4.5:1. Additionally, when these buttons are hovered, the maroon background lightens but still does not meet a contrast ratio of 4.5:1
